main-site/components/adventure-ahead-section/AdventureAheadSection.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PrimaryButton from "../../../shared-ui/components/primary-button/PrimaryButton";
+import { H3 } from "../../../shared-ui/style/typography";
+import { colors } from '../../../shared-ui/style/colors';
+
+import { StyledAdeventureAheadSectionContainer, 
+  StyledParagraph, 
+  StyledSignpost,
+  StyledTextContainer,
+StyledButtonContainer } from "./AdventureAheadSection.styles";
+
+import Signpost from "../../../shared-ui/images/signpost.svg"
```

Comment:
I think you forgot to add the svg to your PR so the build is failing + I can't run this locally ahaha
<img width="1368" alt="Screen Shot 2022-10-04 at 5 09 56 PM" src="https://user-images.githubusercontent.com/59738880/193929558-12ff9d6d-b259-4288-8491-819f03014ac6.png">